.github/workflows/tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           requirements-builder -e "$EXTRAS" --level=${{ matrix.requirements-level }} setup.py > .${{ matrix.requirements-level }}-${{ matrix.python-version }}-requirements.txt
 
       - name: Cache pip
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
I took the liberty of adding dependabot.yml to the PR so that we can automate that chore in the future :smiley: .